api/renderers.py

```diff
@@ -25,10 +25,16 @@ class BrowsableAPIRenderer(renderers.BrowsableAPIRenderer):
     """
     'Custom' Browsable API Renderer
 
-    By setting 'display_edit_forms' to False in the context,
-    you can display the API without forms.
+    By returning an empty rendered HTML form
+    you can display the API quickly, without
+    having to deal with "select" queryset slow-downs.
     """
 
+    def get_rendered_html_form(self, data, view, method, request):
+        if method in ['PUT','POST']:
+            return ""
```

Comment:
This single change resulted in (Easily) a 50x speed-up of the Browsable API.

JSON rendering was never affected.